Source/WebCore/Modules/indexeddb/server/UniqueIDBDatabase.cpp

```diff
@@ -1428,8 +1427,10 @@ void UniqueIDBDatabase::immediateClose()
     // Error out all transactions.
     // Pending transactions must be cleared before in-progress transactions,
     // or they may get started right away after aborting in-progress transactions.
-    for (auto& transaction : m_pendingTransactions)
-        transaction->databaseConnection().deleteTransaction(*transaction);
+    for (auto& transaction : m_pendingTransactions) {
```

Comment:
Is there a chance m_pendingTransactions gets mutated while being iterated.
Should we move m_pendingTransactions to a new pendingTransactions and iterate this new pendingTransactions instead?